modules/ROOT/examples/live-demos/advtemplate-insertionpoint/index.html

```diff
@@ -0,0 +1,40 @@
+<textarea id="advanced-template-insertionpoint">
+<h3>Using the Insertion Point Marker with Advanced Templates (and Merge Tags)<h3>
+
+<h4>To insert a template</h4>
+ <ol>
+  <li>
+    Select <strong>Template…</strong> from the <strong>Insert</strong> menu or<br />
+    select the <strong>Insert template</strong> toolbar button.
+  </li>
+  <li>
+  Select the template to add to the TinyMCE document from the <strong>Templates</strong> dialog that presents.
+    <ol>
+      <li>
+      Click <strong>Without an insertion point marker</strong> to see and select a template that does not use the <code>{{mce-cursor}}</code> insertion point marker.
+      </li>
+      <li>
+      Click <strong>With an insertion point marker</strong> to see and select a template that does use the <code>{{mce-cursor}}</code> insertion point marker.
+      </li>
+    </ol>
+  </li>
```

Comment:
There are no UI components to choose Insertion with/without a point marker. If the marker is present in the template the caret will be placed on it, otherwise not.

Reply:
What about updating this part? We still need to remove the "Click Without/With an insertion point marker ..."